src/renderer/features/discord-rpc/use-discord-rpc.ts

```diff
@@ -77,6 +77,12 @@ export const useDiscordRpc = () => {
                     type: discordSettings.showAsListening ? 2 : 0,
                 };
 
+                if (song?.artistName) {
+                    activity.stateUrl = 'https://www.last.fm/music/' + song.artistName;
```

Comment:
If you have multiple artists, full artist name might not be great (e.g., you _may_ have better results with just the first artist, for example). Album URL would have to be the full thing though